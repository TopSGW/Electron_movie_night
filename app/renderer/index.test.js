'use strict'
/* globals jest, test, expect */

import logger from './mainWindowLogger'
import mockRenderer from 'react-test-renderer'

jest.mock('react-dom', () => {
  return {
    render: (component, root) => {
      const render = mockRenderer.create(component)
      let tree = render.toJSON()
      expect(tree).toMatchSnapshot()
    }
  }
})

test('app renders without crashing', () => {
  require('./index')
})

test('basic logging', () => {
  logger.debug('debug')
  logger.info('info')
  logger.warn('warn')
  logger.error('error')
})
