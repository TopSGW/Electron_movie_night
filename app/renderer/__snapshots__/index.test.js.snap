// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app renders without crashing 1`] = `
<div
  className="sc-EHOje gFrDcL sc-bdVaJa jJjJOT"
>
  <span
    className="sc-bwzfXH dTlhFj"
  >
    Movie
  </span>
  <i
    className="fa fa-anchor sc-htpNat eorCwZ fa-lg"
  />
  <span
    className="sc-bwzfXH dTlhFj"
  >
    Night
  </span>
</div>
`;
